CMakeLists.txt

```diff
@@ -90,3 +90,15 @@ caffe_print_configuration_summary()
 
 # ---[ Export configs generation
 caffe_generate_export_configs()
+
+# uninstall target
```

Comment:
You could try to conform with their style of ```# ---[ uninstall target```. Also, the entire block should be placed in line [88](https://github.com/jasjuang/caffe/blob/d8cebca234e4d8e57b41edd0495e3b3028140d75/CMakeLists.txt#L88).